src/main/kotlin/com/mparticle/kits/MParticleAutopilot.kt

```diff
@@ -91,8 +90,7 @@ class MParticleAutopilot : Autopilot() {
          * @param configuration UrbanAirshipKit configuration.
          */
         fun updateConfig(context: Context, configuration: UrbanAirshipConfiguration) {
-            val editor = context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE)
-                .edit()
+            val editor = context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE).edit()
```

Comment:
some formatting changes here that could be unintended? Not sure what best practices are for Kotlin.

Reply:
Automatic corrections made by running the Kotlin formatter

Reply:
@Mansi-mParticle that's fine, but going forward please make sure to commit formatting changes as a separate commit rather than mixed in with the other changes. It adds significant review overhead as it makes it harder to see which changes are relevant to the PR.

Reply:
Yes, I can do that. My previous PR failed due to Kotlin lint issues, so now I run the Kotlin formatter command before creating a PR. Do we still need to make a separate PR?

Reply:
@Mansi-mParticle it can be the same PR if it's needed to meet PR check requirements, but should be a separate commit so it's easier to filter out